Gemfile

```diff
@@ -5,6 +5,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in superform.gemspec
 gemspec
 
+gem "debug"
```

Comment:
I realize this might suck for your local development workflow, but I'd like to avoid adding new dependencies to this project. I'm hoping there's some way you can still use the `debug` gem for your local workflow or `binding.irb` is enough to get by. If none of those work and you plan on making lots of contributions then I'd consider merging in a separate commit and PR that explains the reason for this dependency to future committers.